crates/cli/src/main.rs

```diff
@@ -0,0 +1,150 @@
+use clap::Parser;
+use clap::Subcommand;
+use gevulot_node::rpc_client::RpcClient;
+use std::net::SocketAddr;
+use std::path::PathBuf;
+
+#[derive(Parser, Debug)]
+#[clap(author = "Gevulot Team", version, about, long_about = None)]
+pub struct ArgConfiguration {
+    /// RPC url of the Gevulot node
+    #[clap(
+        short,
+        long,
+        default_value = "http://localhost:9944",
+        value_name = "URL"
+    )]
+    jsonurl: String,
```

Comment:
```suggestion
    json_url: String,
```
Similar to earlier comment on variable names, I feel like these params could also be formatted so that they are split to words. WDYT?

Reply:
I use this for because - or _ in cli cmd parameters is not idiomatic, so I think we should use jsonurl or url for the paramters. I'll specify the parameter name in the [derive] and use a more idiomatic way to name Rust fields.